- demux: mp4: add missing values for object_type_indication
  2123990073760a6faa34c92cef04d84703a8944a
- packetizer: mpeg4: fix mapping entries for HEVC/VVC
  00ee14f06457032edfd6b68959140a1bc528033a

Index: modules/demux/mp4/mpeg4.h
--- modules/demux/mp4/mpeg4.h.orig
+++ modules/demux/mp4/mpeg4.h
@@ -38,6 +38,12 @@ static inline bool MPEG4_Codec_By_ObjectType(uint8_t o
         case 0x21: /* H.264 */
             *pi_codec = VLC_CODEC_H264;
             break;
+        case 0x23: /* H.265 */
+            *pi_codec = VLC_CODEC_HEVC;
+            break;
+        case 0x33: /* H.266 */
+            *pi_codec = VLC_CODEC_VVC;
+            break;
         case 0x40:
         case 0x41:
             *pi_codec = VLC_CODEC_MP4A;
@@ -99,8 +105,14 @@ static inline bool MPEG4_Codec_By_ObjectType(uint8_t o
         case 0xac: /* Extension Substream containing only LBR */
             *pi_codec = VLC_CODEC_DTS;
             break;
+        case 0xB1:
+            *pi_codec = VLC_CODEC_VP9;
+            break;
         case 0xDD:
             *pi_codec = VLC_CODEC_VORBIS;
+            break;
+        case 0xE1:
+            *pi_codec = VLC_CODEC_QCELP;
             break;
         default:
             return false;
